content/about/board/_index.md

```diff
@@ -81,3 +81,11 @@ Russ's drive and passion is to enable all software engineers to achieve incredib
 {{< board-member name="Maximilian Capraro" image="/images/about/Maximilian_Capraro.jpg" >}}
 Maximilian Capraro is a researcher and doctoral candidate at the Open Source Research Group at Friedrich-Alexander-University Erlangen-Nürnberg. He worked on inner source in research projects with a variety of companies including Black Duck Software, Continental, and multiple Siemens business units. He developed the patch-flow measurement method (for measuring software development collaboration) and the first classification framework for inner source projects and programs. His research interests the quantification of software development collaboration, inner source governance including legal and taxation aspects, and many other things around inner source, open source, and software engineering.
 {{< /board-member >}}
+
+{{< board-member name="Tim Yao" image="/images/about/Tim_Yao.jpeg" >}}
+Tim is a Distinguished Member of Technical Staff and Domain Leader for Open Source Innovation in the Nokia Mobile Network CTO Core and Cloud Innovation team, working on software reuse, InnerSource, open source and cloud robotics topics. Over his nineteen-year career in telecommunications, Tim has held roles in software testing, system engineering, architecture, and procurement. He served six years on the Alcatel-Lucent FOSS Executive Committee. Tim has experience creating grass roots communities within the company and outside of it; he is a volunteer co-Municipal Liaison for National Novel Writing Month.
```

Comment:
Updated bio:
Tim is a Distinguished Member of Technical Staff and the Product Manager for Nokia Continuous Delivery in Nokia Cloud and Network Services. Over his twenty-plus year career in telecommunications, Tim has held roles in innovation, software testing, system engineering, architecture, and procurement. He served six years on the Alcatel-Lucent FOSS Executive Committee. Tim has experience creating grass roots communities within the company and outside of it; he is a volunteer co-Municipal Liaison for National Novel Writing Month.

Reply:
Updated in [fcc2ba6](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/278/commits/fcc2ba6efdde60fcdac1bbba6777fade2cb151cb)